sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  AssessmentTriggerType,
+  ScholasticStandingStatus,
+  StudentAppealStatus,
+} from "../../../database/entities";
+
+export enum AssessmentHistoryStatus {
```

Comment:
I understood that is terms of the UI it does represents the assessment history but I would be more inclined to name this as `StudentAssessmentStatus` because it would be more toward `StudentAssessment` main status. Thoughts?

Reply:
mm, at first I was planning to give assessmnetStatus, then in type, he has an assessment_status, which is irrelevant after your today's PR. but `StudentAssessmentStatus` make sense, i will update
![image](https://user-images.githubusercontent.com/77353155/160462629-768c58d1-f120-40d0-b99f-a3a363fc5d0d.png)

Reply:
Thanks for that. Yes, the `assessment_status` was not named in the most accurate way and it caused issues while creating the table specific for the reassessments. I can see the challenge. Thanks for adjusting it.